tools/ubergen/bin/ubergen.ts

```diff
@@ -7,14 +7,17 @@ import * as ts from 'typescript';
 
 const LIB_ROOT = path.resolve(process.cwd(), 'lib');
 const ROOT_PATH = findWorkspacePath();
+const UBER_PACKAGE_JSON_PATH = path.resolve(process.cwd(), 'package.json');
 
 async function main() {
   console.log(`🌴  workspace root path is: ${ROOT_PATH}`);
-  const libraries = await findLibrariesToPackage();
-  const packageJson = await verifyDependencies(libraries);
-  await prepareSourceFiles(libraries, packageJson);
-}
 
+  const uberPackageJson = await fs.readJson(UBER_PACKAGE_JSON_PATH);
```

Comment:
Set this up as a global variable or method as well instead of passing this into each method.

Reply:
I actually rather keep it this way, because the `[verifyDependencies()' function actually mutates this object](https://github.com/aws/aws-cdk/pull/12545/files?file-filters%5B%5D=.ts#diff-640fff866fbd53269999fced9a5790e0ccbb8d4be1f1a0d77ff6b3d25cf8984fR205).